src/NHSD.BuyingCatalogue.Identity.Api/Certificates/Certificate.cs

```diff
@@ -35,19 +37,56 @@ private static X509Certificate2 Load(ILogger logger, CertificateSettings setting
 
             try
             {
-                var certificate = new X509Certificate2(path, settings.CertificatePassword);
+                byte[] certBuffer = GetBytesFromPem(settings.CertificatePath, PemStringType.Certificate);
```

Comment:
I'm not sure the enum is needed here as you have already made the decision as to which header & footer of the certificate to use. Wouldn't it be better to create two methods to supply the appropriate header and footer to the `GetBytesFromPem` method? This way it wouldn't have to return null (switch statement) & if the `GetBytesFromPem` method requires a header and footer it is clearer. The code could also throw an argum,ent null exception if the header & footer are null as it looks like they shouldn't be.